quic/s2n-quic-crypto/src/one_rtt.rs

```diff
@@ -8,7 +8,6 @@ negotiated_crypto!(OneRttKey, OneRttHeaderKey);
 
 impl crypto::OneRttKey for OneRttKey {
     #[inline]
-    #[must_use]
```

Comment:
Explanation: this attribute has no effect on trait implementations. It's the trait function itself that should use this attribute, and in this case, derive_next_key() does.